src/GDLInterpreter.hpp

```diff
@@ -223,7 +224,8 @@ class CUSTOM_API GDLInterpreter : public antlr::TreeParser, public GDLInterprete
 //             }
 //         return false;
 //     }
-
+ //   static bool IsEnabledGC() { return enable_GC;}
+	static bool IsEnabledGC() { return true;}
```

Comment:
is this intentionally set to "return true"?

Reply:
Yes.  This means, globally, EnabledGC().
  I don't have reason to set this false and am not sure I had a strong logic one way or the other,
so in order to insert feature and let someone else worry about ramifications, I confine it to true.

An operable IsEnabledGC() is kept in objects.cpp that can be happily toggled to no effect.